### PR TITLE
Cfy 3691 make all internal rest calls secure

### PR DESCRIPTION
**Pull request by noakup**

### CODE REVIEW

File: fabric_plugin/tasks.py [hunk 1]
```diff
@@ -460,7 +460,7 @@ def host_string(self):
         if 'host_string' in self.fabric_env:
             host_string = self.fabric_env['host_string']
         else:
-            host_string = self.ctx.instance.host_ip
+            host_string = self.ctx.instance.private_ip
```
Comment (dankilman) [inline fabric_plugin/tasks.py]: keep host_ip

File: fabric_plugin/tests/blueprint/fabric_tasks.py [hunk 2]
```diff
@@ -10,7 +10,10 @@ def test_task_properties(arg):
 
 
 def test_implicit_host_string():
+    # TODO: use a different value, internal_manager_host is canceled
+    # ctx.instance.runtime_properties['expected_host_string'] = \
+    #     ctx.instance.internal_manager_host
     ctx.instance.runtime_properties['expected_host_string'] = \
-        ctx.instance.host_ip
+        ctx.instance.private_ip
```
Comment (dankilman) [inline fabric_plugin/tests/blueprint/fabric_tasks.py]: "

**Comment by nir0s** [comment]
Can we close this?

**Comment by dankilman** [comment]
Most of this PR includes unrelated changes.

**Pull request closed without merging**